tardis/simulation/base.py

```diff
@@ -130,15 +130,15 @@ def __init__(
         iterations,
         model,
         plasma,
-        runner,
+        montecarlo_transport,
         no_of_packets,
         no_of_virtual_packets,
         luminosity_nu_start,
         luminosity_nu_end,
         last_no_of_packets,
         luminosity_requested,
         convergence_strategy,
-        nthreads,
+        #nthreads,
```

Comment:
just remove - don't comment

Reply:
Yes, being a draft PR I hadn't finalized things. It's all done now, please review the current changes.